spec/factories/content_items_factory.rb

```diff
@@ -6,5 +6,14 @@
     sequence(:document_type) { |index| "document_type-#{index}" }
     base_path "api/content/item/path"
     public_updated_at { Time.now }
+
+    factory :content_items_with_org do
+      transient do
```

Comment:
Why are you choosing two organisations here? I would prefer to have only 1, if no big reason for 2.